version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.3.6-SNAPSHOT"
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
what is the reason for the "big" bump?

Reply:
the fiddle integration breaks the API

Reply:
e.g. a `0.3` fiddle directive will no work with `0.4`

Reply:
ok, then the bump makes sense